user_guide_src/source/dbmgmt/forge.rst

```diff
@@ -323,13 +323,14 @@ Class Reference
 
                 Adds a field to the set that will be used to create a table. Usage:  See `Adding Fields`_.
 
-    .. php:method:: addForeignKey($fieldName, $tableName, $tableField[, $onUpdate = '', $onDelete = ''])
+    .. php:method:: addForeignKey($fieldName, $tableName, $tableField[, $onUpdate = '', $onDelete = '', $fkName = ''])
 
         :param    string|string[]    $fieldName: Name of a key field or an array of fields
         :param    string    $tableName: Name of a parent table
         :param    string|string[]    $tableField: Name of a parent table field or an array of fields
         :param    string    $onUpdate: Desired action for the "on update"
         :param    string    $onDelete: Desired action for the "on delete"
+        :param    string    $fkName: Name of foreign key
```

Comment:
It would be nice to have a note near this method description about one not supported driver.

Reply:
Done